### PR TITLE
style guide

### PR DESCRIPTION
**Pull request by chantelle11**
Preview link can be found [here](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/add/feature/style-guide-preview/style/)

# Description
Move style guide information from github to docs page in own /style route

# Screenshots
## style/
Screen Shot 2022-01-31 at 11 28 41 AM

## /style-voice/
Screen Shot 2022-01-31 at 11 29 09 AM

Screen Shot 2022-01-31 at 11 29 21 AM

## /style-headings/

Screen Shot 2022-01-31 at 11 29 39 AM

## /style-links/

Screen Shot 2022-01-31 at 11 30 05 AM

## /style-formatting/

Screen Shot 2022-01-31 at 11 30 28 AM

## /style-code/
Screen Shot 2022-01-31 at 11 30 56 AM

## /style-lists/
Screen Shot 2022-01-31 at 11 31 12 AM

## /style-tables/
Screen Shot 2022-01-31 at 11 31 39 AM

## /style-images/
Screen Shot 2022-01-31 at 11 32 02 AM

### CODE REVIEW

File: jekyll/_config.yml [hunk 1]
```diff
@@ -133,6 +140,13 @@ defaults:
     values:
       lang: "ja"
 
+  -
+    scope:
+      path: ""
+      type: "style"
+    values:
+      lang: "en"
+
```
Comment (rsylvian) [inline jekyll/_config.yml:149]: @chantelle11 This also needs to be put in the other configs file (`_config-stagging` and `_config-prod`)

File: jekyll/_layouts/classic-docs.html [hunk 2]
```diff
@@ -3,21 +3,26 @@
 {% include_cached head.html site=site page=page %}
 <body class="{% if page.page-type == "index" or page.page-type == "homepage" %}homepage{% endif %}">
 {% if jekyll.environment == "production" %}
-  {% include_cached google-tag-manager-noscript.html %}
+  {% include_cached google-tag-manager.html %}
```
Comment (rsylvian) [inline jekyll/_layouts/classic-docs.html:6]: Why is this changed?

File: jekyll/_layouts/classic-docs.html [hunk 3]
```diff
@@ -48,12 +53,12 @@
...
-            <div id="helpful-resources">
-              <h5 class="helpful-heading">Helpful resources</h5>
-              <div class="helpful-indent">
+            <div id="related-resources">
+              <h5 class="related-heading">Helpful resources</h5>
+              <div class="related-indent">
                 <ul class="section-nav">
                   {% for item in page.suggested %}
-                  <li class="helpful-entry helpful-h2">
+                  <li class="related-entry related-h2">
```
Comment (rsylvian) [inline jekyll/_layouts/classic-docs.html:61]: You should not have these changes in this PR. Make sure to rebase master, please 🙏

**Review by rsylvian — CHANGES_REQUESTED** [review]

**Review by rosieyohannan — APPROVED** [review]

**Review by Mayuko-Sa — APPROVED** [review]

File: jekyll/_style/style-code.adoc [hunk 4]
```diff
@@ -0,0 +1,9 @@
+= Using code samples
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+* When using example code snippits, be mindful that people are likely to copy paste them. Always point out, just before the snippet, which, if any, items or values need to be replaced with the reader’s own data – including whether they should replace the surrounding `< >`` or `“ “`.
```
Comment (rosieyohannan) [inline jekyll/_style/style-code.adoc:7]:
```suggestion
* When using example code snippits, be mindful that people are likely to copy paste them. Always point out, just before the snippet, which, if any, items or values need to be replaced with the reader’s own data – including whether they should replace the surrounding `< >` or `“ “`.
```

Comment (rosieyohannan) [inline jekyll/_style/style-code.adoc]:
```suggestion
* When using example code snippits, be mindful that people are likely to copy paste them. Just before the snippet point out which, if any, items or values need to be replaced with the reader’s own data, including whether they should replace the surrounding `< >` or `“ “`.
```

File: jekyll/_style/style-headings.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
+= Headings
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+* Write headings in a logical sequence. If you read out section titles in order, this should present a good storyline for the doc.
```
Comment (rosieyohannan) [inline jekyll/_style/style-headings.adoc:7]:
```suggestion
* Write headings in a logical sequence. If you read section titles in order, the flow should present a good storyline for the doc.
```

File: jekyll/_style/style-headings.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+* Do not skip heading levels – try not to jump from, for example, `h2` to `h4`.
```
Comment (rosieyohannan) [inline jekyll/_style/style-headings.adoc:9]:
```suggestion
* Do not skip heading levels. For example, try not to jump from `h2` to `h4`, missing out `h3`.
```

File: jekyll/_style/style-headings.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+* Do not skip heading levels. For example, try not to jump from `h2` to `h4`, missing out `h3`.
+
+* Try not to have a heading with no paragraph text to accompany it. Add at least an overview or intro to what comes next.
+
+* All headings are in sentence case. This means only the first word and proper nouns are capitalized. 
+** **Good:** CircleCI docs style guide +
+** **Bad:** CircleCI Docs Style Guide
+
+* Jekyll automatically creates an h1 heading from the doc title, so only use h2 and greater in the body of the doc.
```
Comment (rosieyohannan) [inline jekyll/_style/style-headings.adoc:17]:
```suggestion
* Jekyll automatically creates an `h1` heading from the document title, so only use `h2` and greater in the body of the doc.
```

File: jekyll/_style/style-voice.adoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+* No whitelist/blacklist. Instead use safelist/blocklist.
+* No master/slave. Instead use leader/follower or primary/replica depending on the context.
+
+== Abbreviations
+* Do not use the abbreviations "e.g." or "i.e.". Instead, use "for example" and "that is", respectively.
+
+== Punctuation
+* Avoid semicolons and dashes (en or em) to split sentences. Instead, add a new sentence, or use commas and periods.
+* Do not use ampersands (&) as a substitute for the word "and". The ampersand is used in logical notation for the binary operator AND.
+* Don't put a period at the end of a sentence if the sentence ends with a URL, code sample or command. People will be likely to copy and paste and could end up copying the period too.
```
Comment (rosieyohannan) [inline jekyll/_style/style-voice.adoc:29]:
```suggestion
* Do not put a period at the end of a sentence if the sentence ends with a URL, code sample or command. People are likely to copy and paste and could end up copying the period too.
```

File: jekyll/_style/style-voice.adoc [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+* **Good**: Click **Go** and your project will complete.
+* **Bad**: When you click the **Go** button your project should complete.
+
+== Write for anyone and everyone
+Assume technical competence but explain concepts clearly and simply. Some concepts we write about are complex, and outside a non-technical reader's experience, but the flow of CircleCI docs should be accessible to anyone.
+
+== Word choices
+* Steer clear of making assumptions about the complexity of a process. Avoid descriptions like "easy", "easily" or "simply".
+* Avoid ambiguous language such as "should" and "should be". If something is optional, say so, if not make it clear it is an essential step.
+* When specifying a tool or way of performing an action, use "using" instead of "via". The latter is harder on non-English speakers.
```
Comment (rosieyohannan) [inline jekyll/_style/style-voice.adoc:19]:
```suggestion
* When specifying a tool or way of performing an action, use "using" instead of "via". The latter is harder on non-English speakers.
* Avoid contractions, for example "don't", "you're". They make the content harder for non-native english speakers to read.
```

**Comment by rosieyohannan** [comment]
Hey @rsylvian is there work still to do here? I would like to publish this ASAP and then we're iterate on the content as style issues come up. Thanks!

File: jekyll/_style/style.adoc [hunk 10]
```diff
@@ -0,0 +1,14 @@
+---
+description: "Style guide for contributions to CircleCI docs."
+permalink: style/
+page-type: index
+---
+= Welcome to the CircleCI style guide
+:page-layout: classic-docs
+:icons: font
+:toc: macro
```
Comment (chantelle11) [inline jekyll/_style/style.adoc:9]: any adoc that doesn't have a header should set :toc: false

**Review by rsylvian — APPROVED** [review]
